src/Workspaces/Core/Portable/Shared/Extensions/SemanticEquivalence.cs

```diff
@@ -97,6 +125,10 @@ public static bool AreEquivalent(SemanticModel semanticModel, SyntaxNode node1,
             }
         }
 
+        private static bool IsInstanceMemberReference(IOperation operation)
+                => operation is IMemberReferenceOperation memberReferenceOperation &&
+                    memberReferenceOperation.Instance?.Kind == OperationKind.InstanceReference;
```

Comment:
```suggestion
                => operation is IMemberReferenceOperation { Instance: { Kind: OperationKind.InstanceReference } };
```

Reply:
I didn't know you could do that, is there a term for this kind of formatting?

Reply:
yup.  this is 'pattern matching'.  specifically 'property pattern matching' :)